checkov/terraform/checks/resource/aws/ElasticsearchDomainAuditLogging.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class ElasticsearchDomainAuditLogging(BaseResourceCheck):
+    def __init__(self) -> None:
+        name = "Ensure Elasticsearch Domain Audit Logging is enabled"
+        id = "CKV_AWS_317"
```

Comment:
shouldn't this value be different to terraform and cloudformation? I remembered that we do not allow it like that

Reply:
we keep them insync, as long as they are for different runners/frameworks. Easier to manage for users, if you have a mixed IaC setup.